main.py

```diff
@@ -3,5 +1,29 @@
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Drama")
+    Actor.objects.create(first_name="George", last_name="Clooney")
+    Actor.objects.create(first_name="Keanu", last_name="Reeves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+
+    Genre.objects.filter(name="Drama").update(name="Drama")
+
+    Actor.objects.filter(first_name="George",
+                         last_name="Clooney").update(last_name="Clooney")
```

Comment:
The update operation for 'George Clooney' is redundant as it updates the last name to the same value it already has. Consider removing this operation.